### PR TITLE
POL-441 AWS S3 Intelligent Tiering Policy

### PR DESCRIPTION
**Pull request by rshade**
### Description

Creates a policy to identify aws s3 buckets that do not have intelligent tiering enabled. 

### Issues Resolved

n/a

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/aws/s3_storage_policy/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# AWS Bucket Size Check
```
Comment (srpomeroy) [inline cost/aws/s3_storage_policy/README.md:1]: Update name

File: cost/aws/s3_storage_policy/aws_s3_bucket_policy_check.pt [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+  label "Send Email"
+  description "Send incident email"
+  email $param_email
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "aws_oversized_buckets" do
```
Comment (srpomeroy) [inline cost/aws/s3_storage_policy/aws_s3_bucket_policy_check.pt:245]: Change name

File: cost/aws/s3_storage_policy/aws_s3_bucket_policy_check.pt [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+  validate_each $ds_combined_bucket_info do
+    summary_template "{{ len data }} Buckets without intelligent tiering policies found in AWS Account ID: {{with index data 0}}{{ .accountId }}{{end}}"
```
Comment (srpomeroy) [inline cost/aws/s3_storage_policy/aws_s3_bucket_policy_check.pt:247]: We should probably standardize on the syntax/layout of summary templates.
Example: https://github.com/flexera/policy_templates/blob/master/cost/aws/unused_volumes/aws_delete_unused_volumes.pt#L713

File: cost/aws/s3_storage_policy/aws_s3_bucket_policy_check.pt [hunk 4]
```diff
@@ -0,0 +1,268 @@
+name "AWS S3 Bucket Intelligent Tiering Check"
+rs_pt_ver 20180301
+type "policy"
+short_description "This Policy Template scans all s3 buckets and reports if they don't have intelligent tiering enabled. See the [README](https://github.com/flexera/policy_templates/tree/master/cost/aws/s3_storage_policy) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
+long_description ""
+severity "medium"
+category "Cost"
+default_frequency "daily"
+info(
+  version: "2.5",
```
Comment (srpomeroy) [inline cost/aws/s3_storage_policy/aws_s3_bucket_policy_check.pt:10]: Should be 2.0

**Review by srpomeroy — CHANGES_REQUESTED** [review]
A handful of things to tidy up.

**Review by FabriceScoupeFlexera — COMMENTED** [review]
Apart from what Sean pointed out, it looks good to me.
Once those few things have been fixed, I'm happy to approve.

**Review by FabriceScoupeFlexera — APPROVED** [review]
LGTM, but I'm deferring to Sean for final approval as he is the one who requested the changes.

**Review by srpomeroy — APPROVED** [review]

**Pull request merged**